src/attached_probe.cpp

```diff
@@ -758,9 +759,12 @@ void AttachedProbe::load_prog(BPFfeature &feature)
         opts.expected_attach_type = static_cast<::bpf_attach_type>(
             libbpf::BPF_TRACE_ITER);
 
+      // We want to avoid kprobe_multi here because the BPF_TRACE_KPROBE_MULTI
+      // link type does not currently support the `module:function` syntax.
```

Comment:
I don't like the wording too much, what about
```suggestion
      // We want to avoid kprobe_multi when a module is specified because the BPF_TRACE_KPROBE_MULTI
      // link type does not currently support the `module:function` syntax.
```